test/functions/socialFunctionsTest.js

```diff
@@ -0,0 +1,61 @@
+require('../helper.js')
+
+const slug = 'santiment'
+const from = new Date(2019, 0, 1, 0, 0, 0)
```

Comment:
I know we are faking but let's keep `from`, `to` and the response we are requesting with the same dates. Otherwise it can get confusing for the reader. 
This test would serve as documentation, so the easiest way is to use the same dates you are entering in the graphiql request from where you copied the response.

Reply:
Alright, I will make the Dates there